x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -6,59 +6,82 @@
  */
 
 import {
-  EuiInMemoryTable,
   EuiBasicTableColumn,
   EuiLink,
   EuiTitle,
   EuiFlexGroup,
   EuiFlexItem,
   EuiSpacer,
   EuiText,
+  EuiIconTip,
 } from '@elastic/eui';
 import React from 'react';
-import styled from 'styled-components';
 
-import { SummaryRow } from './helpers';
+import type { AlertSummaryRow } from './helpers';
+import * as i18n from './translations';
 import { VIEW_ALL_FIELDS } from './translations';
+import { SummaryTable } from './table/summary_table';
+import { SummaryValueCell } from './table/summary_value_cell';
+import { PrevalenceCellRenderer } from './table/prevalence_cell';
+// import { AddToTimelineCellRenderer } from './table/add_to_timeline_cell';
```

Comment:
consider removing this if it's no longer required

Reply:
Done!